src/myrtlespeech/protos/deep_speech_2.proto

```diff
@@ -19,7 +19,11 @@ message DeepSpeech2 {
       Conv1d conv1d = 1;
       Conv2d conv2d = 2;
     }
-    Activation activation = 3;
+
+    // If true then batch normalization is added after the conv layer.
+    bool batch_norm = 3;
+
+    Activation activation = 4;
```

Comment:
Keep `Activation activation = 3;` and put `bool batch_norm = 4;`. I agree that the batch_norm should go first in the file but we shouldn't change the numbers used in the proto encoding after they are in master as this will break existing protos. From the proto docs:
> each field in the message definition has a unique number. These field numbers are used to identify your fields in the message binary format, and should not be changed once your message type is in use. 
https://developers.google.com/protocol-buffers/docs/proto3#assigning-field-numbers

`master` isn't really 'in use' yet so this is more of a flag for future best-practices.

Reply:
I've just changed it. In the fully_connected proto file I already did it correctly (batch norm after activation field) but for some reason I forgot to do it also for the conv fields.